### DISCUSSION TITLE
Meson findings in quickfix

### DISCUSSION THREAD

**Discussion by tvannahl** [mason-org/mason.nvim#1727] [Q&A] [answered]
Moin,

I've recently switched from my own hand crafted vim/neovim config to lazyvim and thus became a fresh user of meson. What a fantastic experience ❤️ but also so many new integrations so its hard for me to name things.

Unfortunately the errors, warnings, etc. reported by the meson/lsp plugins are missing in the [quickfix](https://vimhelp.org/quickfix.txt.html) list. 

Thus I do have two questions:

#. Is it possible to add these?
#. How are the inline integrated error messages called?

**Comment by ilias777** [comment, marked as answer]
Hi.

To add all the errors, warnings, etc. in the quickfix, you have to send these with [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim). With the command `Telescope diagnostics` you can see all the diagnostics in the Telescope window. Then press `<C-q>` to send these in the quickfix window. After that you can open the quickfix with `:cope` or `:cope` to see them.

Second way is, you can install [trouble.nvim](https://github.com/folke/trouble.nvim). Really good plugin to see all the diagnostics.

I hope this helps.

**Reply by tvannahl** [reply]
Thank you 🙏  - `trouble.vim` really does look amazing.
